src/app/main.cpp

```diff
@@ -52,7 +52,7 @@
 
 #if HAVE_STATIC_QCA_PLUGINS
 #include <QtPlugin>
-Q_IMPORT_PLUGIN( opensslPlugin )
+// Q_IMPORT_PLUGIN( opensslPlugin )
```

Comment:
Review

Reply:
@m-kuhn , cause of crash:

--------- beginning of crash
12-31 23:03:56.555 18204 18239 E AndroidRuntime: FATAL EXCEPTION: qtMainLoopThread
12-31 23:03:56.555 18204 18239 E AndroidRuntime: Process: ch.opengis.qfield_beta, PID: 18204
12-31 23:03:56.555 18204 18239 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: library "libqca-ossl.so" not found: needed by /data/app/~~PXkdAMRHMsvMmkmuj4n_sg==/ch.opengis.qfield_beta-Nuz0-nhYGdz96d8L236v8A==/lib/arm64/libqfield_arm64-v8a.so in namespace clns-4

Reply:
The .so is indeed missing from the APK.